string robot_id
string robot_status
Pose waiting_location